compilation_tests/ctest_no_unique_address.cpp

```diff
@@ -99,19 +99,11 @@ MDSPAN_STATIC_TEST(
 );
 
 MDSPAN_STATIC_TEST(
-  sizeof(stdex::layout_stride<stdex::dynamic_extent, 1, 3>::template mapping<
+  sizeof(stdex::layout_stride::template mapping<
     stdex::extents<42, stdex::dynamic_extent, 73>
-  >) == 2 * sizeof(ptrdiff_t)
+  >) == 4 * sizeof(size_t)
 );
 
-// Fails on MSVC which adds some padding 
```

Comment:
this test is simply not true anymore since strides are now always stored dynamically as proposed.